x-pack/test/fleet_api_integration/apis/epm/install_custom.ts

```diff
@@ -26,7 +26,7 @@ export default function (providerContext: FtrProviderContext) {
       .send({ force: true });
   };
 
-  describe('TESTME Installing custom integrations', async () => {
+  describe('Installing custom integrations', async () => {
```

Comment:
This `TESTME` string in the test name looked like a leftover from someone's `--grep` FTR workflow, so I removed it.